src/pybind/mgr/dashboard/frontend/src/app/ceph/smb/smb.model.ts

```diff
@@ -0,0 +1,14 @@
+export interface SMBCluster {
+  cluster_id: string;
+  auth_mode: string;
```

Comment:
```suggestion
  auth_mode: AuthMode;
```
`auth_mode` will be a string, but it has to either be `user` or `active-directory`. 
There for I think we could give it a custom type for it

Reply:
What can be given in Authmode interface ?
@Pegonzal

Reply:
export interface AuthMode {
  user: 'User';
  activeDirectory: 'active-directory';
}

Reply:
Sorry, I missed this. 
I think that would work, also something like 

`export type AuthMode = 'user' | 'active-directory'`

would do the trick as well